devops/e2e/modulestest.json

```diff
@@ -0,0 +1,44 @@
+[
```

Comment:
Just curious, why `modulestest.json` instead of a workflow matrix?

Reply:
Although you can technically achieve the same, i felt it easier to manage this way, its easier to read and easy to consume outside of workflows as well